python/ray/tests/test_runtime_env_conda_and_pip_2.py

```diff
@@ -33,37 +37,36 @@ def test_install_failure_logging(
         if using_ray_client:
             with pytest.raises(ConnectionAbortedError) as excinfo:
                 ray.init(address, runtime_env=bad_envs["init"])
-            assert bad_packages["init"] in str(excinfo.value)
+                assert bad_packages["init"] in str(excinfo.value)
         else:
             ray.init(address, runtime_env=bad_envs["init"])
-            wait_for_condition(
-                lambda: bad_packages["init"] in capsys.readouterr().out, timeout=30
-            )
```

Comment:
Why get rid of this?  Don't we want to test that if we do ray.init(bad env), the user sees a good error message?

Reply:
We stopped printing runtime env logs, so this wait for condition won't work.

Instead I ensured when tasks are scheduled, they are failing with runtime env error.